templates/firefox-ios.xliff

```diff
@@ -3932,6 +3932,14 @@ Private Tab</source>
         <source>Default Search Engine</source>
         <note>Accessibility label for default search engine setting.</note>
       </trans-unit>
+      <trans-unit id="Settings.Search.Accessibility.LearnAboutSuggestions.v124" xml:space="preserve">
+        <source>Learn more about %@ Suggest</source>
```

Comment:
Seems like we're not using a placeholder in the case of "Firefox Suggest" (see [here](https://transvision.mozfr.org/?recherche=Firefox+Suggest&repo=all_projects&sourcelocale=en-US&locale=fr&search_type=strings) for example). That permits locales to use declensions when needed.